test/test.js

```diff
@@ -99,6 +100,7 @@ describe('fill-pdf', function() {
 
     it('should generate valid data', function() {
       var actual = fillPdf.generateFdf(formData);
+      console.log(actual.toString());
```

Comment:
Don't need this console.log in the test. As soon as it is removed I'll merge PR. Thanks for working with the tests!